docs/running/tardis_widgets/using_widgets.rst

```diff
@@ -0,0 +1,105 @@
+********************
+Using TARDIS Widgets
+********************
+
+This page describes what each TARDIS Widget has to offer and how you can make
+the best use of it. If you're looking for the code to generate widgets, head
+over to `Generating TARDIS Widgets <generating_widgets>`_ section and see the
+notebook in action.
+
+Currently, TARDIS supports the following widgets:
+
+Shell Info Widget
+#################
+
+This widget allows you to explore the chemical abundances in different shells
+of a Supernova ejecta.
+
+.. image:: images/shell-info-widget-demo.gif
+    :alt: Demo of Shell Info Widget
+
+It consists of four interlinked tables - clicking on any row in a table,
+populates data in the table(s) right to it. All these 4 tables, in order, has
+following information:
+
+1. **Shells Data** - Radiative temperature and Dilution Factor (W) of each shell
+(computational partitions) of a Supernova ejecta. Shell numbers in ascending 
```

Comment:
"for each cell...of the model supernova ejecta"

"Shell numbers are given in ascending order, from the innermost shell to the outermost."